src/containers/AppWithClusters/AppWithClusters.tsx

```diff
@@ -18,32 +13,21 @@ import {ExtendedTenant} from './ExtendedTenant/ExtendedTenant';
 export interface AppWithClustersProps {
     store: Store;
     history: History;
-    getLogsLink?: GetLogsLink;
-    getMonitoringLink?: GetMonitoringLink;
-    getMonitoringClusterLink?: GetMonitoringClusterLink;
     userSettings?: YDBEmbeddedUISettings;
     children?: React.ReactNode;
 }
 
-export function AppWithClusters({
-    store,
-    history,
-    getLogsLink,
-    getMonitoringLink = getMonitoringLinkDefault,
-    getMonitoringClusterLink = getMonitoringClusterLinkDefault,
-    userSettings,
-    children,
-}: AppWithClustersProps) {
+export function AppWithClusters({store, history, userSettings, children}: AppWithClustersProps) {
     return (
         <App store={store} history={history} userSettings={userSettings}>
             <AppSlots.ClusterSlot>
                 {({component}) => {
                     return (
                         <ExtendedCluster
                             component={component}
-                            getLogsLink={getLogsLink}
-                            getMonitoringLink={getMonitoringLink}
-                            getMonitoringClusterLink={getMonitoringClusterLink}
+                            getLogsLink={uiFactory.getLogsLink}
```

Comment:
I didn't refactored additional props themselves, it's could be done further, these changes are to merge with current major and to not do another